src/running-policies.md

```diff
@@ -0,0 +1,230 @@
+# Running Policies
+
+Running policies is by far the most common operation that a Kubernetes
+operator will perform. You can configure as many policies as you want,
+targeting any kind of resource and operation on those
+resources. Policies that will run in a Kubernetes cluster is not a
+static setting and might evolve over time: new policies can be
+enforced, existing policies modified, updated or removed...
+
+The recommended deployment model of Chimera on top of Kubernetes is to
+use the [`chimera-controller`
+project](https://github.com/chimera-kube/chimera-controller).
+
+With the `chimera-controller` project you will be able to manage
+admission policies in a very easy, safe and straightforward way.
+
+## Deploy the `chimera-controller`
+
+First of all, you will need an operative Kubernetes cluster, with a
+`kubectl` context pointing to it.
+
+1. Deploy `cert-manager`:
+
+    ```console
+    $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+    $ kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+    ```
+
+1. Deploy the `chimera-controller` after `cert-manager` is ready:
+
+    ```console
+    $ kubectl apply -f https://raw.githubusercontent.com/chimera-kube/chimera-controller/v0.0.1/config/generated/all.yaml
+    $ kubectl wait --for=condition=Available deployment --timeout=2m -n chimera-controller-system --all
+    ```
+
```

Comment:
I would add something like:

> This will deploy the different components of the Chimera stack and will define a new type of Kubernetes Custom Resource used to declare Chimera policies.